resources/views/pagination-url.blade.php

```diff
@@ -18,12 +18,12 @@
 
         @if($paginator->onFirstPage())
             <div class="flex items-center button-generic button-disabled">
-                <div class="flex items-center">@svg('chevron-left', 'h-3 w-3')<span class="hidden lg:flex lg:ml-2">Previous</span></div>
+                <div class="flex items-center"><span class="hidden lg:flex lg:ml-2">Previous</span></div>
```

Comment:
don't remove that, this pagination component is used by more projects and removing that will need to be discussed first

Reply:
also it's unrelated to dark mode

Reply:
Alrighty, gonna put them back, just followed the designs ; https://gyazo.com/e44e2a2cdbeecc6213d3f0a60624f4c9

Reply:
@Highjhacker I think the chevron you put back might be in the wrong place based on the pr diff